osf/apps.py

```diff
@@ -3,6 +3,12 @@
 from django.apps import AppConfig as BaseAppConfig
 from django.db.models.signals import post_migrate
 from osf.migrations import update_permission_groups
+from django.core.management import call_command
+from api.base import settings
+
+
+def create_cache_table(*args, **kwargs):
```

Comment:
See if this works moved into `osf/migrations/__init__.py` to keep consistent with the `update_permissions_groups` call